plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -300,7 +300,7 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
     protected void renameTable(ConnectorSession session, String catalogName, String schemaName, String tableName, SchemaTableName newTable)
     {
         if (!schemaName.equals(newTable.getSchemaName())) {
-            throw new TrinoException(NOT_SUPPORTED, "Table rename across schemas is not supported in PostgreSQL");
+            throw new TrinoException(NOT_SUPPORTED, "This connector does not support renaming tables across schemas");
```

Comment:
Could you please doublecheck if exception message corresponds to condition correctly.
I don't understand how case equality correlates with `across schemas`.